CHANGELOG.md

```diff
@@ -6,14 +6,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
```

Comment:
```suggestion

## Changed
- The default Postgres server version is incremented to 10.16 from 10.15. [Postgres 10.16](https://www.postgresql.org/docs/10/release-10-16.html)
   [cyberark/conjur-oss-helm-chart#147](https://github.com/cyberark/conjur-oss-helm-chart/pull/147)

```
The changelog entry can go above the line of the last release, in the "Unreleased" section